react/personal_project_vite/src/App.css

```diff
@@ -4,7 +4,7 @@
   padding: 2rem;
```

Comment:
Thank you for utilizing responsive units of measurement. `rem` and `em` are actually units of measurement for text size and utilizing them for different purposes may cause some unexpected behavior. It may be a good call to utilize `vmin, vmax, vh, vw, or %` for padding and other measurements through out your Front End design.